### PR TITLE
bundle library using rollup

### PR DESCRIPTION
**Pull request by alexanbj**
This pull request bundles the library using Rollup.

This is because it will make it possible to consume the library without using a bundler. In the previous beta you were required to use a bundler, since the eventemitter library we rely on was CJS, while the lib itself was ESM. With these changes we create a single ESM file, which should be easy to import.

### CODE REVIEW

**Comment by trygve-lie** [comment]
This outputs the same structure of the files but as esm right? Its not bundled into one file.

**Comment by alexanbj** [comment]
> This outputs the same structure of the files but as esm right? Its not bundled into one file.

No, it's bundled to a single file

**Comment by trygve-lie** [comment]
Can we have it output the initial structure?

File: rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+
+export default {
+    input: 'src/index.js',
+    output: {
+        file: 'dist/index.js',
```
Comment (stipsan) [inline rollup.config.js]:
```suggestion
        dir: 'dist',
```

File: rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        format: 'esm',
+    },
+    plugins: [resolve(), commonjs()],
```
Comment (stipsan) [inline rollup.config.js]:
```suggestion
    plugins: [resolve(), commonjs()],
    preserveModules: true,
```

**Pull request merged**